### PR TITLE
Add copernicus services and framework tags

### PR DESCRIPTION
**Pull request by Kampen1988**
Urgent EDC public collections updates. Add *copernicus services* tag, as well as tags for *CAMS*, *CLMS* and *C3S* for respective collections.

### CODE REVIEW

File: collections/copernicus-dem.yaml [hunk 1]
```diff
@@ -21,6 +21,7 @@ Contact: https://forum.sentinel-hub.com/c/euro-data-cube/25
 Provider: "[Copernicus](https://copernicus.eu/)"
 ManagedBy: "[Sentinel Hub](https://www.sentinel-hub.com/)"
 Tags:
+  - copernicus services
```
Comment (dthiex) [inline collections/copernicus-dem.yaml:24]: Are you sure this is part of copernicus services? It was not in Grega's list and I could not find a explicit source saying this. If you are sure feel free to resolve and merge.

**Review by dthiex — APPROVED** [review]

File: collections/copernicus-dem.yaml (hunk 1 above)
Comment (Kampen1988) [reply collections/copernicus-dem.yaml]: Wasn't sure about that one, I'll leave it out.

**Review by dthiex — APPROVED** [review]
Looks all good now. We can still add the tag to the cop-dem later if we find that it's part of Copernicus services.

File: collections/copernicus-dem.yaml (hunk 1 above)
Comment (zcernigoj) [reply collections/copernicus-dem.yaml]: Does it still need the `RegistryEntryLastModified` change after that?

**Pull request merged**